src/app/(auth)/(navigationsBarLayout)/problems/page.tsx

```
@@ -35,111 +46,116 @@ const ProblemsList = () => {
-                  className="cursor-pointer"
-                  alt="close"
-                  width={16}
-                  height={16}
-                  onClick={() => setDifficulty('')}
+              </FilterSelect>
+              <FilterSelect>
+                <label className="text-base">난이도 </label>
+                <Select
+                  className="w-full"
+                  title="난이도"
+                  option={difficultyOptions}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)